entity-types/infra-awsbackup/summary_metrics.yml

```diff
@@ -0,0 +1,18 @@
+providerAccountName:
+  title: AWS account
+  unit: STRING
+  tag:
+    key: providerAccountName
+region:
+  title: Region
+  unit: STRING
+  tag:
+    key: aws.awsRegion
+numberOfBackupJobsRunning:
+  goldenMetric: NumberOfBackupJobsRunning
```

Comment:
Please use the same name from goldenMetric, including capitalized letters:

`numberOfBackupJobsFailed`